libs/components/indicators/src/lib/modules/key-info/key-info-value.component.scss

```diff
@@ -0,0 +1,21 @@
+@use 'libs/components/theme/src/lib/styles/mixins' as mixins;
+@use 'libs/components/theme/src/lib/styles/variables' as vars;
+
+// Apply .sky-font-display-3 styles for default theme.
+sky-key-info-value {
+  font-family: vars.$sky-font-family-condensed;
+  color: var(--sky-text-color-default);
+  font-weight: 600;
+  font-size: 22px;
+}
+
+@include mixins.sky-theme-modern {
+  // Apply .sky-font-display-3 styles for modern theme.
+  sky-key-info-value {
+    color: var(--sky-color-text-default);
+    font-family: var(--sky-font-family-primary);
+    font-size: var(--sky-font-size-display-3);
+    font-weight: var(--sky-font-style-display-3);
+    line-height: var(--sky-font-line_height-display-3);
```

Comment:
I tried to get `@extend` to work, but it was creating too much specificity, overriding the user-provided class on the element (there's the standard `.sky-font-display-3` selector, but also the modern-theme selector). 

I also attempted to create new mixins for the shared styles, but it was adding a bit too much complexity for my taste. 

I remember Paul bringing up a viable option to just copy/paste the `.sky-font-display-3` styles into the component since the values (for modern theme) are custom properties, so that seemed like the best option if code complexity is the concern.